src/app/pages/storage/volumes/manager/vdev/vdev.component.html

```diff
@@ -93,6 +93,7 @@
       </button>
     </div>
   </div>
-  <div *ngIf="error" [style.color]="'red'" [innerHTML]="error" class="datainfo"></div>
+  <div *ngIf="error" [innerHTML]="error" class="sizeMessage red"></div>
+  <div *ngIf="showDiskSizeError" [innerHTML]="diskSizeErrorMsg" class="sizeMessage orange"></div>
```

Comment:
There is no html in `diskSizeErrorMsg`, you can just do `{{ diskSizeErrorMsg }}`